### ISSUE TITLE
Publish flow: increase prominence of setting a post as "sticky"

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#63386]
Finding the option to make a post sticky has just gotten substantially harder in the last GB release. Similar to revisions https://github.com/WordPress/gutenberg/issues/62041, this feels like something to surface more or at least place in a spot that might be easier to find. Curious why we got rid of the toggle on/off for sticky posts @WordPress/gutenberg-design 

Previous in 6.5:

https://github.com/WordPress/gutenberg/assets/26996883/518c7b26-cebc-4410-bff6-04fc54d9999f

Current in 6.6:

https://github.com/WordPress/gutenberg/assets/26996883/289a230b-0f88-4a55-aaef-f05a48468182

Current in GB 18.7.1:

https://github.com/WordPress/gutenberg/assets/26996883/af8f6ffd-4dfc-495d-8ace-a754e7cf7d19

**Comment by t-hamano** [comment]
For the latest Gutenberg, the setting was moved to the status popover via #62782.

One reason for this is that this setting might be closely related to status and visibility. For example, in the classic editor, this setting is in "Visibility".

![visibility-closed](https://github.com/WordPress/gutenberg/assets/54422211/a61911e4-129d-4878-8c46-ec4585422d51)

![visibility-opened](https://github.com/WordPress/gutenberg/assets/54422211/29ea07cd-10eb-471f-92e5-96b3519510f7)

**Comment by jameskoster** [comment]
I wonder if there's a way to make it more findable while still managing prominence?

One option could be to expand this UI to include things like stickiness and order. Stickiness could be on by default, order off. When toggled on the options appear as rows in the summary panel.

<img width="577" alt="Screenshot 2024-07-11 at 10 07 45" src="https://github.com/WordPress/gutenberg/assets/846565/b9f98caf-3026-422d-8745-dbd6e9dce695">

That way folks can manage the prominence in the UI based on their usage.

**Comment by afercia** [comment]
> That way folks can manage the prominence in the UI based on their usage.

It's an interesting point. I would like to point out that before the Post Summary panel redesign, users already had the ability to adapt the Post panel based on their usage and flows. In fact, most of the setting were grouped into the collapsible panels. The expanded / collapsed state of these panels is stored in the `preferencesStore` so that the UI 'remembers' which panels should be rendered expanded by default and which ones collapsed by default. 

That mechanism provided a good way for users to have what they need more prominent based on their own flows. For example, as an user that doesn't use much often the featured image, I had the choice to keep its panel always collapsed. Same for other panels.

Now, with the new Post Summary panel, the ability to manage prominence of these settings is lost. The new Post Summary panel is based on a series of assumptions that completely removed the ability for users to have the settings they need prominent and collapse the ones they don't need. For example, as as an user that doesn't use much often the featured image, I would like the featured image to not be always there at the top of the panel because I really don't need it and it's just a piece of UI that stands in my way unnecessarily.

Overall, I'd think the redesign of the Post Summary panel should be reconsidered to take more into consideration users feedback received since the redesign and, more importantly, it should be _user tested_.

**Comment by Marc-pi** [comment]
yep, this need a better place / be promeninent like it was (the toggle), this feature is usefull
would be nice also to make it available to other CPT as well
